static/upload.css

```diff
@@ -0,0 +1,22 @@
+body {
+    background-color: #0B0C10;
```

Comment:
I don't know about this color...

Reply:
I wanted to make a dark themed website. Any recommendations?